single_level.py

```diff
@@ -90,22 +90,69 @@ def makeRow(resource):
 		else:
 			row.append("false")
 	
-	print row
+	print row	
 
 def main():
-	print "Creating a csv"
+	#User input to refine functionality of script 
+	print ""
+	print "Welcome to DACSspace!"
+	print ""
```

Comment:
Another way of getting a line break is to use `\n`, so 
`print "Welcome to DACSspace!\n"`
is a slightly cleaner way of writing
```
print "Welcome to DACSspace!"
print ""
```
(This boils down to personal preference, though!)